duolingo.py

```diff
@@ -73,18 +98,20 @@ def _make_dict(self, keys, array):
         return data
 
     def _compute_dependency_order(self, skills):
-        """ Add a field to each skill indicating the order it was learned
-            based on the skill's dependencies. Multiple skills will have the same
-            position if they have the same dependencies.
+        """
+        Add a field to each skill indicating the order it was learned
+        based on the skill's dependencies. Multiple skills will have the same
+        position if they have the same dependencies.
         """
         # Key skills by first dependency. Dependency sets can be uniquely
         # identified by one dependency in the set.
-        dependency_to_skill = MultiDict([(skill['dependencies_name'][0]
-                                        if len(skill['dependencies_name']) > 0 else '', skill)
-                                        for skill in skills])
+        dependency_to_skill = MultiDict([
+            (skill['dependencies_name'][0]
+             if len(skill['dependencies_name']) > 0 else '', skill)
+            for skill in skills])
```

Comment:
for this one as well, I personally would prefer it if the `MultiDict` data is all on the right of the `=`

something like this or similar

```
dependency_to_skill = MultiDict([(skill['dependencies_name'][0]
                                   if len(skill['dependencies_name']) > 0
                                   else '',
                                  skill)
                                 for skill in skills])

```